main.go

```diff
@@ -38,6 +48,19 @@ type ConfigsModel struct {
 	ManualMerge      string
 }
 
+func init() {
```

Comment:
move the `ConfigsModel` initializer (`createConfigsModelFromEnvs`) and it's instance functions (`print`, `validate`) here.